app/models/base_item.rb

```diff
@@ -14,6 +14,7 @@
 #
 
 class BaseItem < ApplicationRecord
+  include Itemizable
```

Comment:
Sorry, the issue had some bad wording - this has to be added to the `Item` class, not `BaseItem`.